### DISCUSSION TITLE
Option to ignore/cancel Hysteresis if the temperature peak ends before response time ends.

### DISCUSSION THREAD

**Discussion by TheDeeGee** [Rem0o/FanControl.Releases#2581] [Ideas]
I would like to see Hysteresis being expanded upon.

For example:

My CPU Fan Curve is set stay 0 RPM until it hits 50C, and Hysteresis is set to respond to 3C changes after 5 seconds. So when my CPU sits at 48C and briefly peaks to 52C the Hysteresis kicks in 5 seconds later, but often before the Response timer ends the CPU is already back to 47-48C, so it doesn't have to spin the fans anymore, but still does because it's a simple delay.

So it would be nice to have an option which tells Hysteresis the temperature increase needs to "remain" for an X amount of seconds before it kicks in, instead of a pure delay.

**Comment by Rem0o** [comment]
> which tells Hysteresis the temperature increase needs to "remain" for an X amount of seconds before it kicks in

That's exactly how response time + hysteresis works already.  If it goes back down to the previous temp before the delay, it will not do anything.

**Comment by TheDeeGee** [comment]
Then i guess something is wrong with my config or the sensors on my motherboard are wonky.
